plugin/workspace_symbol.py

```diff
@@ -10,9 +10,6 @@
 
 class SymbolQueryInput(sublime_plugin.TextInputHandler):
 
-    def validate(self, txt: str) -> bool:
-        return txt != ""
-
     def placeholder(self) -> str:
         return "Symbol"
```

Comment:
Maybe placeholder could indicate that empty string is allowed. Something like `Symbol (leave empty to request all symbols)`